### ISSUE TITLE
Rename default branch to `main`

### ISSUE THREAD

**Issue by sharifelgamal** [kubernetes/minikube#13184]
This will require auditing all our CI in github actions and jenkins to see what currently relies on the current `master` name.

**Comment by medyagh** [comment]
good idea, lets dedicate one day after 1.25.0 release to do this all over our automations

**Comment by sharifelgamal** [comment]
This is still something we want to do.

**Comment by Ritikaa96** [comment]
I see this hasn't been done yet as default branch is still `master`. Is there specific reason behind trying to change it? just curious.
